### PR TITLE
web: Add merchant creation workflow skeleton

### PR DESCRIPTION
**Pull request by backspace**
This includes some components from prepaid card issuance, I’ve updated the
relevant issues to highlight that.

### CODE REVIEW

File: packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+          message: `Hello, nice to see you!`,
+        }),
+        new WorkflowMessage({
+          author: cardbot,
+          message: `To receive payment through Card Pay, you need to create a merchant account.
+          All you need is to choose a name for your merchant account and sign a transaction to create your merchant on the ${c.layer2.fullName}.`,
+        }),
+        new NetworkAwareWorkflowMessage({
+          author: cardbot,
+          message: `Looks like you've already connected your ${c.layer2.fullName} wallet, which you can see below.
```
Comment (lukemelia) [inline packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts:37]:
```suggestion
          message: `Looks like you’ve already connected your ${c.layer2.fullName} wallet, which you can see below.
```

**Review by lukemelia — APPROVED** [review]
🙆

**Review by Aierie — APPROVED** [review]
Looks good to me! 

More of a design comment - it seems odd that the message and header introduces this step as "giving a merchant a name", when we giving it more (color, id, assigning its manager) + creating it.

image

File: packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts (hunk 1 above)
Comment (backspace) [reply packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts]: agreed, thanks. This came from a copypaste and I see it in a couple of other places, I’ll open another PR for that.

**Comment by backspace** [comment]
> More of a design comment - it seems odd that the message and header introduces this step as "giving a merchant a name", when we giving it more (color, id, assigning its manager) + creating it.

I agree. I’ll make a subissue about this.

**Pull request merged**